.ci/generate_bear_requirements.py

```diff
@@ -155,18 +206,32 @@ def get_pip_requirements(requirements):
     pip_reqs, npm_reqs, gem_reqs = (
         get_all_requirements(get_all_bears(bear_dirs)))
 
-    requirements = {}
-    requirements['overrides'] = 'coala-build.yaml'
-    requirements['pip_requirements'] = get_pip_requirements(pip_reqs)
-    requirements['npm_requirements'] = get_npm_requirements(npm_reqs)
+    requirements = CommentedMap()
     requirements['gem_requirements'] = get_gem_requirements(gem_reqs)
+    requirements['npm_requirements'] = get_npm_requirements(npm_reqs)
```

Comment:
why re-order these lines?

Reply:
To keep diff of the generated file to a minimum.